baseline.yaml

```
@@ -649,6 +649,26 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-13
+    maturity_level: 1
+    category: Documentation
+    criteria: |
+       The project documentation MUST include a 
+       descriptive statement about how the project 
+       is supported/maintained and what is not supported. 
```

```suggestion
       The project documentation MUST include a 
       descriptive statement about the scope and 
       duration of support. 
```

I'm bringing this back up one more time because I think it's more clear and direct. But I also won't block on it.